<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screencap</class>
 <widget class="QMainWindow" name="Screencap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Part Video Recording Station</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnConfig">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStation">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>Recording Station</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnStart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnStop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUSN">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>USN</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Powered By Magnetivity</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUSNText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>A234ABC123</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMessage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledRed">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>90</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: red;
    border-radius: 18px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledGreen">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: green;
    border-radius: 18px;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
